### ISSUE TITLE
Flutter [channel=dev] compilation error, 'sortBy' is defined in multiple extensions and neither is more specific

### ISSUE THREAD

**Issue by shemhazai** [JonasWanke/rrule#14]
**Describe the bug**
Using the rrule package with Flutter on channel dev stops the compilation with the following error:

```
../../Library/flutter/.pub-cache/hosted/pub.dartlang.org/rrule-0.1.1/lib/src/codecs/text/encoder.dart:383:57: Error: The method 'sortBy' is defined in multiple extensions for 'List<T>' and neither is more specific.
 - 'List' is from 'dart:core'.
Try using an explicit extension application of the wanted extension or hiding unwanted extensions from scope.
    final negative = where((e) => key(e) < 0).toList()..sortBy(key);
                                                        ^^^^^^
../../Library/flutter/.pub-cache/hosted/pub.dartlang.org/basics-0.4.0+1/lib/list_basics.dart:93:8: Context: This is one of the extension members.
  void sortBy(Comparable Function(E) sortKey) {
       ^^^^^^
../../Library/flutter/.pub-cache/hosted/pub.dartlang.org/collection-1.15.0-nullsafety.5/lib/src/list_extensions.dart:192:8: Context: This is one of the extension members.
  void sortBy<K extends Comparable<K>>(K Function(E element) keyOf,
       ^^^^^^
```

**Environment:**

[✓] Flutter (Channel dev, 1.24.0-10.2.pre, on Mac OS X 10.15.7 19H15 darwin-x64, locale en-PL)
    • Flutter version 1.24.0-10.2.pre at /Users/shemhazai/Library/flutter
    • Framework revision 022b333a08 (16 hours ago), 2020-11-18 11:35:09 -0800
    • Engine revision 07c1eed46b
    • Dart version 2.12.0 (build 2.12.0-29.10.beta)

rrule: version 0.1.1

The problem does not occur on stable & beta flutter channels.

**Comment by IoanaAlexandru** [comment]
This is now a problem on the beta channel as well.

**Comment by JonasWanke** [comment]
This is now fixed as part of [v0.1.2](https://github.com/JonasWanke/rrule/releases/tag/v0.1.2).
